packages/frontend/src/admin-pages/components/admin/pages/custom/CustomPagesEdit.vue

```diff
@@ -182,7 +182,7 @@
 </template>
 
 <script>
-import { getContentKeys, getSettingKeys } from './../../../../helpers/content'
+import { getSettingKeys, getContentKeys } from '../../../../helpers/content.js'
```

Comment:
does this need `.js`?

Reply:
Jira added that bit for me... good catch

Reply:
I removed all of the `.js's`